submissions/phpslonuk/Popup/css/style.css

```diff
@@ -0,0 +1,222 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 1170px;
+  margin: 0 auto;
+}
+
+header {
```

Comment:
would be better to add the class to the header and move styles to this class, could you please fix it